codegen/templates/code/controller/multicluster_reconcilers.gotmpl

```diff
@@ -0,0 +1,89 @@
+// Definitions for the multicluster Kubernetes Controllers
+package multicluster
```

Comment:
Can we add some suffix related to the group, so each one doesn't have the same package name. Possibly the version?

Reply:
On second thought i'm a little reluctant to do this because then we have directory structures like `group.solo.io/v1/controller_v1/multicluster_v1` which reads redundant IMO, and since that doesn't guarantee uniqueness when there are two groups with a v1 I don't think it's buying us much over leaving the user to alias.

Reply:
@joekelley why not just leave it in `controller` package?

Reply:
this was done to deal with an import cycle, i'll see if there's a way to bring it back into the controller pkg

Reply:
```
package github.com/solo-io/skv2/codegen/render_test
	imports github.com/solo-io/skv2/codegen/test/api/things.test.io/v1/controller
	imports github.com/solo-io/skv2/pkg/multicluster
	imports github.com/solo-io/skv2/pkg/generated/kubernetes/core/v1/controller
	imports github.com/solo-io/skv2/pkg/multicluster
```

Reply:
gonna fix the cycle

Reply:
will resolve the cycle before merging